src/Esia/Signer/CliSignerPKCS7.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Esia\Signer;
+
+use Esia\Signer\Exceptions\SignFailException;
+use Psr\Log\LoggerAwareTrait;
+
+
+class CliSignerPKCS7 extends SignerPKCS7
```

Comment:
It's better to keep Signers independent and don't share logic between them.

Reply:
but what about certPath, checkFilesExists, getRandomString, and urlSafe?

Reply:
Let's create  AbstractSignerPKCS7 where we can store common logic.